config/locales/en.yml

```diff
@@ -969,9 +969,9 @@ en:
         hint: Enter 0 if this does not apply
         instruction: Enter amount
       pension:
-        label: Pension
-        hint: Include State Pension, workplace and personal pensions. Enter 0 if this does not apply
-        instruction: Enter amount
+        label: Pensions
+        hint: Include the total value of any pensions, State Pension, workplace and personal pensions. Enter 0 if this does not apply.
+        instruction: Enter total value of any pensions'
```

Comment:
There's a rogue `'` at the end of this line and likewise line 1010 - `pensions` by itself is a plural (as it should be). Adding an apostrophe at the end makes it a possessive plural, i.e. it would start to mean "owned by the pensions".